spec/extractors/stanford_organizations_spec.rb

```diff
@@ -32,7 +32,7 @@
       end
 
       it 'prints out the exception' do
-        expect { extractor.each {} }.to output("Error: #{error_message}\n").to_stderr
+        expect { extractor.each {} }.to output(/Uh oh!/).to_stderr
```

Comment:
Can you say more about how this change fits here?

Reply:
Sure. Previously errors caught by stanford organizations extractor where logged with `warn` which output to STDERR. Those were switched to be sent to the logger, which outputs them in a log formatted string. Since that log formatted string might change (e.g., the timestamp), just looking for the presence of the message.

Reply:
OK! Makes sense.